packages/design-system-scripts/gulp/docs/generatePages/uniquePages.js

```diff
@@ -11,8 +11,9 @@ function uniquePages(pages) {
   pages.forEach((page) => {
     if (routes[page.reference]) {
       if (routes[page.reference].source.path.match(/node_modules/)) {
-        // We override pages that come from `node_modules`
+        // We override existing pages that come from `node_modules` with pages from child design system
```

Comment:
// We override pages that come from `node_modules` with pages from the child design system

Is more accurate. This `if` statement for overriding pages that already are in the `routes` map, but I'm not sure if thats important. Adding the `existing` word could be confusing